downloader/downloadGCS_test.go

```diff
@@ -0,0 +1,70 @@
+package downloader_test
+
+import (
+	"google.golang.org/appengine/aetest"
+	"os"
+	"testing"
+
+	"github.com/m-lab/annotation-service/downloader"
+	"github.com/m-lab/annotation-service/parser"
+)
+
+func TestInitilizationTableIPv4(t *testing.T) {
```

Comment:
This function and the below function TestInitilizationTableIPv6() has too much common code.

Can you try to refactor and reduce the dup code?